.github/scripts/check-modified-subtree.sh

```diff
@@ -1,46 +1,123 @@
 #!/usr/bin/env bash
 
-set -o errexit
-set -o nounset
+#
+#   Prints a list of errors for any dependencies that have been edited.
+#   ➞ Dependencies should be edited in their respective repositories
+#
 
-# This script will print GitHub errors when files from subtrees have been edited.
-# Those files should be edited in their respective repositories.
+echo 'Checking for any dependency changes.'
 
-MASTER_REVISION=$(git rev-list --first-parent origin/master | head -n 1)
-DIFFED_FILES=$(git diff --name-only "$MASTER_REVISION")
+set -o errexit # Exit for any error
+set -o nounset # Error when referencing unset vars
 
-FAILED=false
 
-if echo "$DIFFED_FILES" | grep -qP "^gamemode/modules/fpp/pp/"; then
-    echo "::error::Files from Falco's Prop Protection have been edited. Please submit a PR to https://github.com/fptje/falcos-Prop-protection instead!"
-    FAILED=true
-fi
+commit=$(
+    git rev-list                        \
+        --first-parent origin/master    \
+        --max-count=1
+)
 
-if echo "$DIFFED_FILES" | grep -qP "^gamemode/libraries/fn.lua"; then
-    echo "::error::The fn library has been edited. Please submit a PR to https://github.com/fptje/GModFunctional instead!"
-    FAILED=true
-fi
+files=$(
+    git diff        \
+        --name-only \
+        "$commit"
+)
 
-if echo "$DIFFED_FILES" | grep -qP "^gamemode/libraries/mysqlite/mysqlite.lua"; then
-    echo "::error::The MySQLite library has been edited. Please submit a PR to https://github.com/fptje/MySQLite instead!"
-    FAILED=true
-fi
 
-if echo "$DIFFED_FILES" | grep -qP "^gamemode/libraries/simplerr.lua"; then
-    echo "::error::The Simplerr library has been edited. Please submit a PR to https://github.com/fptje/simplerr instead!"
-    FAILED=true
-fi
+#   Template for the GitHub summary
 
-if echo "$DIFFED_FILES" | grep -qP "^gamemode/libraries/sh_cami.lua"; then
-    echo "::error::The CAMI library has been edited. Please submit a PR to https://github.com/glua/CAMI instead!"
-    FAILED=true
-fi
+template="
+    > [!WARNING]
+    > Files from **[{project}]({repository})** have been modified!  
+    > ➞ Changes should only be made in it's repository
```

Comment:
"it's" should be "its", because the apostrophe goes when talking about possession.

Though perhaps a different wording altogether could explain it better, e.g.:

> These files are in the DarkRP repository, but they originally come from {url}. The changes should be applied there instead.

Reply:
Fixed the typo.